src/AvTranscoder/frame/Frame.cpp

```diff
@@ -78,15 +78,18 @@ void Frame::initAVPacket()
 
 void Frame::copyAVPacket( const AVPacket& avPacket )
 {
-#if LIBAVCODEC_VERSION_INT > AV_VERSION_INT(55, 56, 108)
-	av_copy_packet( &_packet, &avPacket );
-#elif LIBAVCODEC_VERSION_INT > AV_VERSION_INT(54, 56, 0)
-	av_copy_packet( &_packet, const_cast<AVPacket*>( &avPacket ) );
-#else
+#if AVTRANSCODER_FFMPEG_DEPENDENCY
+	#if LIBAVCODEC_VERSION_INT > AV_VERSION_INT(55, 56, 108)
+		av_copy_packet( &_packet, &avPacket );
+		return;
+	#elif LIBAVCODEC_VERSION_INT > AV_VERSION_INT(54, 56, 0)
+		av_copy_packet( &_packet, const_cast<AVPacket*>( &avPacket ) );
+		return;
+	#endif
+#endif
```

Comment:
Seems strange to use returns instead of "#else"...

```
#if AVTRANSCODER_FFMPEG_DEPENDENCY && LIBAVCODEC_VERSION_INT > AV_VERSION_INT(55, 56, 108)
```